frontend/src/pages/Community.vue

```diff
@@ -271,6 +281,7 @@ export default {
       pageSize: this.pageSize,
     })
     this.updateTransactions(0)
+    this.tabIndex = 1
```

Comment:
Is this the activation of the first tab?

Reply:
done